sphinx/templates/quickstart/conf.py_t

```diff
@@ -53,9 +53,11 @@ extensions = [
 {%- endfor %}
 ]
 
+{% if dot != '_' -%}
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['{{ dot }}templates']
```

Comment:
I also removed `templates_path` by default. This is very similar to the case of `html_static_path` to me.